packages/loader/container-loader/src/container.ts

```diff
@@ -203,7 +205,8 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         return PerformanceEvent.timedExecAsync(container.logger, { eventName: "Load" }, async (event) => {
             return new Promise<Container>((res, rej) => {
                 const version = request.headers?.[LoaderHeader.version];
-                const pause = request.headers?.[LoaderHeader.pause];
+                // always load unpaused with pending ops
+                const pause = pendingOps !== undefined ? false : request.headers?.[LoaderHeader.pause];
```

Comment:
Same comment as above - should not use "ops" terminology at this level.

Mostly philosophical discussion: :)
We could un-pause container to consume enough ops for local state rehydration to finish, then pause (if asked by caller) and return container in exactly same state as if there was no local state saved. This will create our API more consistent.
That said, this points to somewhat tied knowledge between layers on what this content and what is required to rehydrate it. I.e. Container (Loader) knows that in order for Runtime to consume this state, it need to process ops till certain point in time. I think it's fine (in a sense that we "return" container to same state), but if IRuntime did not need it, Countainer could have used some other (faster) mechanism of loading container at same (last known) sequence number.
Not sure we can do about it, other than document it well across layers, i.e. expectations and assumptions on what each layer is supposed to do, and a little bit on why.